contrib/snap/fwupdmgr.wrapper

```diff
@@ -1,2 +1,4 @@
 #!/bin/sh
+LD_LIBRARY_PATH="$SNAP/usr/local/lib${LD_LIBRARY_PATH:+:}${LD_LIBRARY_PATH-}"
```

Comment:
What is being installed into `/snap/usr/local/lib/` now?  I don't believe fwupd does, and I would think that all the dependent libraries for this snap are in `/snap/usr/lib`.

Reply:
Might not be needed. I do not remember why I needed that.